data/campaigns/Eastern_Invasion/scenarios/17b_Weldyn_Besieged.cfg

```diff
@@ -347,7 +349,7 @@
             id=Mal-Ravanal
             name= _ "Mal-Ravanal"
             profile=portraits/mal-ravanal.png
-            hitpoints=80
+            hitpoints=80 # Full amount; I considered having him start with less but it doesn't make sense for him to start injured
```

Comment:
Don't know if this comment is necessary.

Reply:
removed in 33a48a5